src/AzureIoTHub.Portal/Client/Pages/DeviceModels/DeviceModelListPage.razor

```diff
@@ -37,7 +37,7 @@
                     <MudText Typo="Typo.h6">Device Models</MudText>
                     <MudSpacer />
                     <MudIconButton Icon="@Icons.Material.Filled.Refresh" Size="Size.Medium" OnClick="LoadDeviceModels" Class="ma-2"></MudIconButton>
-                    <MudIconButton Icon="@Icons.Material.Filled.Add" Size="Size.Medium" OnClick="AddDeviceModel" Class="ma-2"></MudIconButton>
+                           <a href="/device-models/new"><MudFab Color="Color.Secondary" Icon="@Icons.Material.Filled.Add" Size="Size.Medium"  id="newDeviceButton" /></a>
```

Comment:
This moves the MudIcon button to a HTML tag.
Why ? is there other options to keep the MudTag instead ? and us the AddDeviceModel Event?